tooling/mocks.yml

```diff
@@ -0,0 +1,26 @@
+# To execute this docker-swarm yml file use `docker stack deploy -c mocks.yml mocks`
+
+version: '3.7'
+
+networks:
+  private:
+    driver: overlay
+
+services:
+  repository:
+    image: httpd:2.4
```

Comment:
Can we switch to [NGINX](https://www.nginx.com) as it is more [non-blocking reactive](https://www.nginx.com/blog/thread-pools-boost-performance-9x/) HTTP server. See https://hub.docker.com/_/nginx.

Reply:
This would save much time analysing why Apache is so slow...

Reply:
I will configure nginx in separate PR